Developer Community Code of Conduct.md

```diff
@@ -0,0 +1,45 @@
+# Code of Conduct
+
+## Purpose
+A primary goal of Miro is to be inclusive to the largest number of participants, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe, and welcoming environment for all— regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended – a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct outlines our expectations for all those who participate in our Miro developer community, as well as the consequences for unacceptable behavior. Your participation here is also subject to the [Miro Community Guidelines](https://community.miro.com/community-welcome-guide-14/miro-community-guidelines-28) and the [Miro Terms of Use](https://community.miro.com/site/terms).
```

Comment:
Do we want to call it `Miro developer community` or `Miro platform community` to be more inclusive with people using the platform but that are not necessarily developers?

Reply:
Good point. Updated!